scripts/github-actions-setup

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env bash
 set -euox pipefail
 
-apt-get update
-apt-get install -y \
-    libvirt-daemon-system \
-    qemu-kvm \
-    vagrant
+curl -fsSL https://apt.releases.hashicorp.com/gpg | gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg
+echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/hashicorp.list
```

Comment:
A small nit. On Debian, the `/etc/apt/keyrings` directory would be for user-managed files.

This is not a blocker.